pubsub/google/cloud/pubsub_v1/subscriber/_consumer.py

```diff
@@ -242,10 +242,10 @@ def _blocking_consume(self):
                 # case, break out of the while loop and exit this thread.
                 _LOGGER.debug('Clean RPC loop exit signalled consumer exit.')
                 break
-            except KeyboardInterrupt:
-                self.stop_consuming()
             except Exception as exc:
-                self.active = self._policy.on_exception(exc)
+                recover = self._policy.on_exception(exc)
```

Comment:
Note that if `isinstance(exc, KeyboardInterrupt)` it will get handled in the same way (since it isn't one the retry-able exceptions)

Reply:
That is fine, but can you put a comment to that effect for clarity?

Reply:
@lukesneeringer I actually don't see a reason. As @shileiyu [noted][1], a `KeyboardInterrupt` is impossible to trigger out of the `MainThread`.

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/issues/4463#issuecomment-348001746